app/services/digital_links_creator.rb

```diff
@@ -0,0 +1,26 @@
+class DigitalLinksCreator
```

Comment:
This should also be in the spree namespace (means also relocating the file to `app/services/spree/digital_links_creator.rb`